go

```diff
@@ -34,7 +34,7 @@ def_command :update_gems, 'Update Ruby gems' do |gems = []|
 end
 
 def_command :test, 'Execute automated tests' do |args = []|
-  exec_cmd "bundle exec rake test #{args.join ' '}"
```

Comment:
Yeah, as it stands in the PR, it works, but @mbland suggested (in person)
that that change should be reverted so it still evokes bundle/rake.

On Fri, Nov 6, 2015 at 1:34 PM, Moncef Belyamani <notifications@github.com>
wrote:

> In go <https://github.com/18F/about_yml/pull/29#discussion_r44171146>:
>
> > @@ -34,7 +34,7 @@ def_command :update_gems, 'Update Ruby gems' do |gems = []|
> >  end
> >
> >  def_command :test, 'Execute automated tests' do |args = []|
> > -  exec_cmd "bundle exec rake test #{args.join ' '}"
>
> The change you made is working for me. What happens when you run ./go test
> ?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/18F/about_yml/pull/29/files#r44171146>.
>



-- 
Alison Rowland
Technical Lead, https://fbopen.gsa.gov | 18F
202-317-0124